tests/bigdata_test.cpp

```diff
@@ -0,0 +1,127 @@
+  puts("per year, outselling IBM PC compatibles, Apple computers,");
+  puts("and the Atari 8-bit family of computers. Sam Tramiel, a");
+  puts("later Atari president and the son of Commodore's founder,");
+  puts("said in a 1989 interview, When I was at Commodore we");
+  puts("were building 400,000 C64s a month for a couple of");
+  puts("years. In the UK market, the C64 faced competition from");
+  puts("the BBC Micro, the ZX Spectrum, and later the Amstrad CPC");
+  puts("464. but the C64 was still the second-most-popular");
+  puts("computer in the UK after the ZX Spectrum. The Commodore");
+  puts("64 failed to make any impact in Japan, as their market");
+  puts("was dominated by Japanese computers, such as the NEC");
+  puts("PC-8801, Sharp X1, Fujitsu FM-7, and MSX.");
+  puts("");
+  puts("Part of the Commodore 64's success was its sale in");
+  puts("regular retail stores instead of only electronics or");
+  puts("computer hobbyist specialty stores. Commodore produced");
+  puts("many of its parts in-house to control costs, including");
+  puts("custom integrated circuit chips from MOS Technology. In");
+  puts("the United States, it has been compared to the Ford Model");
+  puts("T automobile for its role in bringing a new technology to");
+  puts("middle-class households via creative and affordable");
+  puts("mass-production. Approximately 10,000 commercial software");
+  puts("titles have been made for the Commodore 64, including");
+  puts("development tools, office productivity applications, and");
+  puts("video games. C64 emulators allow anyone with a modern");
+  puts("computer, or a compatible video game console, to run");
+  puts("these programs today. The C64 is also credited with");
+  puts("popularizing the computer demoscene and is still used");
+  puts("today by some computer hobbyists. In 2011, 17 years after");
+  puts("it was taken off the market, research showed that brand");
+  puts("recognition for the model was still at 87%.");
+}
+
+void big_bss()
+{
+    static size_t values[MAX_SAMPLES];
+    static size_t sorted[MAX_SAMPLES]; 
+    static size_t ref[MAX_SAMPLES];
+
+    printf("bigger %lx %d\n", MAX_SAMPLES, 0x1919);
+    printf("1 %lx\n", values);
+    printf("2 %lx\n", sorted);
+    printf("3 %lx\n", ref);
+
+    memset(values, 0, sizeof(values));
+    memset(sorted, 0, sizeof(sorted));
+    memset(ref, 0, sizeof(ref));
+}
+
+int get_len()
+{
+  return 100;
+}
+
+int calc_func2(int x, int y)
+{
+    printf("calc-func\n");
+    int i = x;
+    int j = y;
+    return i + j;
+}
+
+int throw_func1(void)
+{
+  printf("throw-func\n");
+  int x = calc_func2(13, 19);
+  return x;
+}
+
+void big_stack_func(void)
+{
+  printf("big-stack-func\n");
+  char sz[0x4000] ="";
+  strcpy(sz,"hello");
+
+  int num = get_len();
+    
+  {
+      int scopes[num];
+      int temps[num];
+
+      throw_func1();
+  }
+}
+
+TEST(Aarch64MinGW, BigDataTest) 
+{
+    printf("main\n");
```

Comment:
Those `printf`s are useless for the gtest tests. On the other hand, I'd expect a series of `EXPECT_*` calls at the end of this function (fixture method). I am affraid that without them, the tests would pass even if there would be an error.

Reply:
Agree. As it was mentioned in another comment it will be done in second iteration.